### DISCUSSION TITLE
Adding a New Field to Certificate

### DISCUSSION THREAD

**Discussion by mahajanmahesh935** [Sunbird-Ed/Community#755] [General Discussion]
Hello,

@pallakartheekreddy

I need assistance with adding a new field to the certificate beyond the default fields (recipientName, trainingName, issuanceDate).

We attempted the following approaches:

Passing a new field in the certificate SVG as {{credentialSubject.recipientOrg}}, but it didn’t work.
Including the org variable in the certificate creation API, which also didn’t yield the desired results.
Could someone please guide us on how to properly add a custom field (e.g., recipientOrg) to the certificate? Any insights or documentation links would be greatly appreciated.

 @komalm @paritshivani @Abhilash-Kishor @vinod303

**Comment by paritshivani** [comment]
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20
Please guide us here. We need to fulfil clients requirement ASAP

**Comment by paritshivani** [comment]
Hello falks!
Can anyone help us here? is it anyhow possible to change or add fields to certificate?
